### ISSUE TITLE
Collecting response latency

### ISSUE THREAD

**Issue by qianmiao90** [lookit/ember-lookit-frameplayer#134]
**Is your feature request related to a problem? Please describe.**
I am frustrated that children's response latency was not collected. I wonder if children's response accuracy and response could be recorded when they were completing the sort-like game. For example, when children are instructed to sort a Black face with a positive picture together, their response accuracy and latency could be recorded. 

**Is your feature request related to a particular frame?**
Nope, it requires creating a new frame.

**Describe your idea or solution**
https://app-prod-03.implicit.harvard.edu/implicit/Launch?study=/user/mqian/raceibt//raceibt.expt.xml
[https://app-prod-03.implicit.harvard.edu/implicit/Launch?study=/user/mqian/raceibt//raceibt.expt.xml](url)
Here is an example of the task running on Project Implicit.

**Comment by kimberscott** [comment]
Timestamps *are* recorded for most events on Lookit frames, e.g. when each of the options and the next button are clicked in https://lookit.github.io/ember-lookit-frameplayer/classes/Exp-lookit-dialogue-page.html (see 'Events'). You might play around with that to see how close it'd be to what you want, and/or take a look at the existing planned frames in https://github.com/lookit/ember-lookit-frameplayer/issues/72. 

Happy to consider new frames more specialized for response latency - if you could sketch out how you'd want the frame to work, that'd be most helpful, and then other people can think through whether that would satisfy their needs too. (I know you're showing an example of a task you want it to support. What I mean is giving the general description of how the frame should look and work on Lookit - what should be customizable, what events should be recorded, etc. Essentially trying to provide the same information as is shown in the Lookit frame docs, and then we make something that corresponds to that documentation :)

**Comment by qianmiao90** [comment]
@kimberscott 
Sure. Thanks for specifying your questions. 
Here is what my task looks like: Children view a Black face or a White face, appearing **randomly** in the middle of the screen. They are instructed to touch either a happy emoji or a sad emoji appearing randomly at the bottom left or right, depending on the face they see in the middle. For one set of pairings, children are told to click the happy emoji when they see a Black face and to click the sad emoji when they see a White face (**"Incongruent pairing")**, whereas for the other set of pairings they are told to click the sad emoji when they see a Black face and to click the happy emoji when they see a White face (**"congruent pairing"**).
 
Let me take one trial in "incongruent pairing" for an example (see below Figure 1), a Black face is presented in the middle, children need to click the happy emoji on the bottom right. After children's click, feedback will be given: for the correct click, the trial will advance; for the incorrect click, a red "x" will appear (see below Figure 2) and correction is needed for trials to move forward. 
 
This is the data I need for each trial:
•	The type of pairing (congruent or incongruent pairing)
•	Trial number
•	The name of the face image in the middle
•	The name of emoji on the bottom left
•	The name of emoji on the bottom right
•	The emoji children clicked (happy or sad)
•	The response time from the moment a face appears to the moment of children's click on the emoji. 
![image](https://user-images.githubusercontent.com/62067435/78707012-0890c680-78de-11ea-8ec7-07598ab19e66.png)
Figure 1
![image](https://user-images.githubusercontent.com/62067435/78707054-147c8880-78de-11ea-9df7-b7343175db88.png)
Figure 2

**Comment by kimberscott** [comment]
Just for anyone else interested in proposing frames - this is still a description of a particular task, not a proposed more general frame. But it does give me a good idea of what you'd need.

Essentially almost all of this is already doable using the exp-lookit-story-page I linked + randomization, and in fact is a pretty expected usage. You would have all of the data you've listed. The one additional piece is that it sounds like it'd be helpful to show the top picture at a precise time (after loading). 

I've added an item to #72 for it so there'll be an easy-to-use frame for similar reaction time experiments :)
